### PR TITLE
Rename metaversefile -> totum

### PR DESCRIPTION
**Pull request by yaserOSource**
ONE OF MANY

metaversefile -> totum

metaversefileApi -> totumApi

.metaversefile -> .totum

All Scenes have been set up to point to ALL forked repositories for testing purposes, this will be reverted to the webaverse repo on merge!

cd packages/totum and checkout hash 8ab8356b36fe8f8eb290c6b2db35cf86bd3bd5f6

QA Checklist:

Test each scene, try out all the items, can you see every single object in the scn list?
Go through FTUX, and interact with everything.

totum repo PR:

https://github.com/webaverse/totum/pull/41

### CODE REVIEW

File: src/Header.jsx [hunk 1]
```diff
@@ -479,16 +479,92 @@ export default function Header({
...
-              open={open}
-              setOpen={setOpen} 
-              toggleOpen={toggleOpen} 
-              panelsRef={panelsRef} 
-              wearActions={wearActions} 
-              previewCanvasRef={previewCanvasRef}
-              game={game}
-            />
-            <World
+            <Tab
```
Comment (avaer) [inline src/Header.jsx:482]: Why is this changed?

**Review by avaer — CHANGES_REQUESTED** [review]
Was there a bad merge?

File: src/Header.jsx (hunk 1 above)
Comment (yaserOSource) [reply src/Header.jsx:482]: Will fix, will revert this to main branch

Comment (yaserOSource) [reply src/Header.jsx:482]: It's the linting tool

**Comment by yaserOSource** [comment]
> Was there a bad merge?

yes there was

**Comment by yaserOSource** [comment]
PR updated

**Comment by yaserOSource** [comment]
I will test it out some more before passing it through QA

File: deadcode/editor.js [hunk 2]
```diff
@@ -2558,14 +2558,14 @@ const loadHash = async hash => {
...
-  const u = `${storageHost}/ipfs/${hash}/.metaversefile`;
+  const u = `${storageHost}/ipfs/${hash}/.totum`;
   loadedObject = await world.addObject(u);
   loadedObject.name = loadedObject.contentId.match(/([^\/]*)$/)[1];
 };
 const mintNft = async () => {
   const hash = await uploadHash();
   console.log('upload nft', hash);
-  window.location.href = `https://webaverse.com/mint?hash=${hash}&ext=metaversefile`;
+  window.location.href = `https://webaverse.com/mint?hash=${hash}&ext=totum`;
```
Comment (avaer) [inline deadcode/editor.js:2568]: I don't think this part is right. We should not be renaming the actual file IMO. This is literally the file type, which can stay as `metaversefile`.

Comment (yaserOSource) [reply deadcode/editor.js:2568]: got it ok, no problem
